doc/extensibility/dataflow_best_practices.md

```diff
@@ -1,13 +1,14 @@
 # Dataflow Best Coding Practices
 These are suggestions to help avoid some common pitfalls when using Dataflow blocks in CPS.
 
-## Use Slim Dataflow blocks 
-You should always use the Slim version blocks provided by CPS, unless you need advanced capabilities that are not supported (like processing multiple works at the same time). See [Dataflow Examples](dataflow_example.md) for more information.
+## Use Slim Dataflow blocks
+You should always use the Slim Datwflow blocks provided by CPS, unless you need advanced capabilities that are not supported (like processing multiple works at the same time). See [Dataflow Examples](dataflow_example.md) for more information.
 
-## Expose `BroadcastBlock` via API
-You should typically expose `BroadcastBlock` via API instead of TransformBlock, etc
+## Expose only `BroadcastBlock` via API
+API should typically expose Broadcast blocks (`BroadcastBlockSlim` or `BroadcastBlock`) via API instead of other types of blocks (e.g. TransformBlock, etc)
```

Comment:
```suggestion
APIs should typically expose Broadcast blocks (`BroadcastBlockSlim` or `BroadcastBlock`) via API instead of other types of blocks (e.g. `TransformBlock`, etc)
```